reco_utils/dataset/wikidata.py

```diff
@@ -153,8 +153,9 @@ def read_linked_entities(data):
         data (json): dictionary with linked pages
 
     Returns:
-        list: List of liked entityIDs
-        list: List of liked entity names
+        list, list:
```

Comment:
Do we know that this is the Google style and not what we had before? I can't figure it from [this](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google).

Reply:
yeah, it is not clear from that file. This format was found by @YanZhangADS. I personally think it is more clear than what we have before, but happy to hear other views